macro/run_simScript.py

```diff
@@ -382,6 +382,8 @@
  primGen.SetTarget(0., 0.)
  Nuagegen = ROOT.NuageGenerator()
  Nuagegen.EnableExternalDecayer(1) #with 0 external decayer is disable, 1 is enabled
+
+ #CAMM - This is broken now, need info from dedicated SND geo...
```

Comment:
@antonioiuliano2 Can we drop the NuageGenerator? I don't remember anyone using it in the last decade, and I'm pretty sure it wouldn't work right now anyway.